markups/cmark_render.py

```diff
@@ -0,0 +1,38 @@
+# vim: ts=8:sts=8:sw=8:noexpandtab
+
+# This file is part of python-markups module
+# License: BSD
+# Copyright: (C) Shengjing Zhu, 2016
+
+from CommonMark import HtmlRenderer
+from CommonMark.common import escape_xml
+
+
+class CommonMarkRender(HtmlRenderer):
```

Comment:
Upstream CommonMark-py keeps functions the same as official implements of CommonMark in Javascripts. And in [commonmark.js](https://github.com/jgm/commonmark.js), the documents says highlight is a exercise to the user... So I don't think they will add highlight to their code.

When using this in Retext, functions like scroll sync which adds `posmap` to the HTML, I think we need derive the HTMLRender again in Retext. So exposing the `CommonMarkRender` is needed. Any good idea?